### PR TITLE
Combine existing JSON and change log documentation into new markdown files

### LINKED ISSUE #28: Document JSON config file parameters
The parameters for the deprecated FIF config files were already well documented in `docs/UsersGuide.html`. We can probably map much of this same documentation onto the corresponding parameters in the JSON file, but I expect there will be new fields to document as well. I think it's better to add this info to the README file, also.

### PR DESCRIPTION
**Pull request by cstephen**
Closes #28.

This PR combines the still-relevant documentation from two existing versions of HTML documentation. The documentation has also been repackaged as new markdown files in the `docs` directory after we decided it was easier to maintain markdown files instead of HTML going forward.

The two versions of HTML documentation I worked from are:
  - The very old documentation in the `docs/UsersGuide.html` file of the `main` branch.
  - The not-as-old but still outdated documentation stored in the `gh-pages` branch and hosted at http://ua-snap.github.io/alfresco/

Specifically, this PR:
- Adds (almost) all missing JSON config parameter descriptions to the JSON config documentation, and moves this documentation into a new `docs/JSONConfig.md` file. Most of the parameter descriptions were taken from the `docs/UsersGuide.html` version of the documentation that described the parameters in the context of FIF files, but the descriptions also apply to their corresponding parameters in the newer JSON config files.
- The old change log in `ChangeLog` (for 1.x versions of ALFRESCO) has been combined with the change log section from http://ua-snap.github.io/alfresco/ (for 2.x versions of ALFRESCO) into a new `docs/ChangeLog.md`.
- The burn severity algorithm information from http://ua-snap.github.io/alfresco/ has been moved into `docs/BurnSeverity.md`.
- The references section from http://ua-snap.github.io/alfresco/ has been moved into `docs/References.md`.
- Historical fire input documentation has been moved out of `README.md` into `docs/HistoricalFires.md` to be consistent with how we are now using the `docs` subdirectoy, and to keep `README.md` lightweight.
- Removes the outdated documentation files I worked from:`ChangeLog` and`docs/UsersGuide.html`. I'm also planning to delete the `gh-pages` branch that hosts http://ua-snap.github.io/alfresco/ once this PR is merged.

### CODE REVIEW

File: docs/ChangeLog.md [hunk 1]
```diff
@@ -0,0 +1,354 @@
+﻿# ALFRESCO Change Log
+
+## 2.2.1
+- Fixed value index of vegetation GeoTIFF description tag.
```
Comment (brucecrevensten) [inline docs/ChangeLog.md:4]: It's great that you're keeping up the changelog!

**Review by brucecrevensten — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -2,7 +2,13 @@
...
+
+The fire regime is simulated stochastically and is driven by climate, vegetation type, and time since last fire (Rupp et al. 2000a, 2007). ALFRESCO employs a cellular automaton approach, where an ignited pixel may spread to any of the eight surrounding pixels. ‘Ignition’ of a pixel is determined using a random number generator and as a function of the flammability value of that pixel. Fire ‘spread’ depends on the flammability of the receptor pixel and any effects of natural firebreaks including non-vegetated mountain slopes and large water bodies, which do not burn.
+
+Version 1.0.1 can operate at any time step and pixel resolution, however the current model calibration and parameterization was conducted at an annual time step and 1 km2 pixel resolution. A 30 m2 calibration and parameterization is currently underway. Other model developments include refined tundra transition stages, fire suppression effects on fire size, simulated fire severity patterns and fire severity effects on successional rates and trajectories.
```
[the remark this answers is not in the document]
Comment (cstephen) [reply README.md]: Paragraph removed in 7f4fe6e097d327371a78462a7ea4f5950c6a1460.

[the remark this answers is not in the document]
Comment (cstephen) [reply README.md]: Paragraph removed in 7f4fe6e097d327371a78462a7ea4f5950c6a1460.

File: docs/HistoricalFires.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
+# Using historical fire data
+
+Historical fire polygon data can be downloaded from the [Maps/Imagery/Geospatial page of the AICC website](https://fire.ak.blm.gov/predsvcs/maps.php) as a file geodatabase (see "AlaskaFireHistory_Polygons_1940_*"). Polygons from each year of the file geodatabase can be extracted, rasterized, and used as historical fire inputs for ALFRESCO.
+
+The following Bash command will created GeoTIFFs from the AICC fire history polygons geodatabase for each year from 1950 to 2021, for example:
```
[the remark this answers is not in the document]
Comment (cstephen) [reply docs/HistoricalFires.md]: Fixed in 7f4fe6e097d327371a78462a7ea4f5950c6a1460.

**Pull request merged**